app/server/docker/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -eu
+
+PATH_CONFIG=/app/src/resources/config.json
+echo "Waiting for config at $PATH_CONFIG"
+while [ ! -e $PATH_CONFIG ]; do
+  sleep 1
+done
+
+sleep 1
+npm run express
```

Comment:
Is this a safe way of running things in prod, or should we add another ticket for running only the built version?

Reply:
It's not. We should build and use pm2 to serve. Can do that on this branch